mappings/net/minecraft/util/OffsetDoubleList.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS cmt net/minecraft/util/OffsetDoubleList
+	FIELD a oldList Lit/unimi/dsi/fastutil/doubles/DoubleList;
+	FIELD b offset D
+	METHOD <init> (Lit/unimi/dsi/fastutil/doubles/DoubleList;D)V
+		ARG 1 oldList
+		ARG 2 offset
+	METHOD getDouble (I)D
+		ARG 1 position
+	METHOD size ()I
```

Comment:
Does this class implement the `java.util.Collection` interface?

Reply:
In yarn, we do not mark methods that cannot be obfuscated with proguard as deobfuscated. As a result, this should not be included.

Reply:
oh, we're not supposed to? I've been doing that lol

Reply:
asie told me in pr reviews before.

Reply:
Yeah. Don't.

I'll clean this up later but don't